src/components/molecules/post/index.tsx

```diff
@@ -19,6 +33,13 @@ const MoleculesPost = ({post, author}: MoleculesPostType) => {
         <span className="font-bold">@{author.username}</span>
         <span className="text-sm text-slate-400"> · {relTime(post.createdAt)}</span>
         <p>{post.content}</p>
+        {user?.id === author.id && (
+          <button
+            onClick={() => mutate({id: post.id})}
```

Comment:
```suggestion
            onClick={onClickMutate}
```
JSX内に関数を書いてしまうと冗長（可読性が下がってしまう）ので関数化して渡してあげましょう！